src/DayColumn.js

```diff
@@ -140,6 +153,45 @@ class DaySlot extends React.Component {
     );
   }
 
+  renderAvailabilities = () => {
+    let {
+      availabilities,
+      availabilityComponent,
+      availabilityWrapperComponent: AvailabilityWrapper,
+      availabilityStartAccessor,
+      availabilityEndAccessor,
+      min,
+      date,
+    } = this.props;
+    let AvailabilityComponent = availabilityComponent;
+    let styledAvailabilities = getStyledAvailabilities({
+      availabilities, availabilityStartAccessor, availabilityEndAccessor, min, totalMin: this._totalMin, date
+    });
+
+    return styledAvailabilities.map(({availability, style}, index) => {
+      let { height, top } = style;
+
+      return AvailabilityWrapper && (
```

Comment:
In what situation is `AvailabilityWrapper` falsy? This check is odd to me. Isn't there a default `AvailabilityWrapper` component if it's not explicitly passed in by the parent of the calendar?

Reply:
Yeah, totally unnecessary check. Thanks!